crypto/node/siv_ctr.js

```diff
@@ -59,11 +60,11 @@ var Decrypt = async function (Key, Nonce, Ciphertext, AdditionalData) {
   let CiphertextLength = Ciphertext.byteLength;
   let cipher = crypto.createCipheriv(
     'AES-128-CTR',
-    new Uint8Array(Key).slice(32, 48),
-    new Uint8Array(Ciphertext).slice(CiphertextLength - TagSize, CiphertextLength)
+    Key.slice(32, 48),
+    Ciphertext.slice(CiphertextLength - TagSize, CiphertextLength)
   );
   let Plaintext = await new Promise(function (resolve, reject) {
-    var output = new Buffer([]);
+    var output = Buffer.from([]);
```

Comment:
worth having output be converted back to a typed array?

Reply:
What is the suggestion here? Output is expected to be a buffer.

Reply:
before this function returns, it should convert back - so that the return value is a uint8array, the same as what would happen from the subtle implementation.

Reply:
Ok sure. I'll run the e2e tests against this change to ensure it doesn't break anything.

Reply:
I think it makes more sense to do this in the mraebox, since thats the external api.

Reply:
Done.